pom.xml

```diff
@@ -137,6 +137,18 @@
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
+            <dependency>
+                <groupId>org.apache.camel.springboot</groupId>
+                <artifactId>camel-mock-starter</artifactId>
+                <version>${spring-camel.version}</version>
+                <scope>test</scope>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.camel</groupId>
+                <artifactId>camel-test-spring-junit5</artifactId>
```

Comment:
This is not necessary - both explicit declarations may be skipped here, as both are covered by `camel-spring-boot-dependencies` bom.